### ISSUE TITLE
[Feat] Zen Theater Mode - Non-Fullscreen fullscreen

### ISSUE THREAD

**Issue by Perseus333** [Pedro-Gregorio/TubeMod#97]
I will try to work on this but I'm putting this here in case someone wants to give it a go as well.

It would be a feature that would make the theater mode view act as a tabbed full screen mode. Meaning that all the space in the tab (not screen) would be the video).

**Comment by Perseus333** [comment]
> I'd suggest renaming this, something like Full Window mode or Zen Mode because there is a Theater mode already and it would cause confusion.

Yeah, the intent is to replace theater mode with this mode. (probably)

**Comment by ItzELECTR0** [comment]
I'd highly advise against that. I use the current theater mode as is and prefer it over YouTube's default player. This extension is all about using YouTube in any way you want to, so taking away a feature to replace it with something else wouldn't really be wise.
Instead of replacing theater mode, implement this as a new one. Another new feature to make YouTube's experience more personal.

**Comment by Perseus333** [comment]
The thing is that you need some view to replace, you cannot simply create another view unless you can tie it to a key bind or something.

**Comment by ItzELECTR0** [comment]
Well, why not tie it to a UI button in the extension's menu, in that case? Or even better, add that UI button straight in the video player next to the other view buttons.
